docs/persistent_compilation_cache.md

```diff
@@ -17,6 +17,7 @@ import jax.numpy as jnp
 jax.config.update("jax_compilation_cache_dir", "/tmp/jax_cache")
 jax.config.update("jax_persistent_cache_min_entry_size_bytes", -1)
 jax.config.update("jax_persistent_cache_min_compile_time_secs", 0)
+jax.config.update("jax_persistent_cache_enable_xla_caches", "all")
```

Comment:
Maybe use the working one until the option "all" is fixed?

Reply:
Thanks, done.